packages/notifications/docs/07-upgrade-guide.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Upgrading from v2.x
+---
+
+> If you see anything missing from this guide, please do not hesitate to [make a pull request](https://github.com/filamentphp/filament/edit/3.x/packages/notifications/docs/07-upgrade-guide.md) to our repository! Any help is appreciated!
+
+## New requirements
+
+- Laravel v9.0+
+- Livewire v3.0+
+
+## Upgrading automatically
+
+The easiest way to upgrade your app is to run the automated upgrade script. This script will automatically upgrade your application to the latest version of Filament, and make changes to your code which handle most breaking changes.
+
+```bash
+composer require filament/upgrade:"^3.0" --dev
+vendor/bin/filament-v3
+```
+
+Make sure to carefully follow the instructions, and review the changes made by the script. You may need to make some manual changes to your code afterwards, but the script should handle most of the repetitive work for you.
+
+Finally, you must run `php artisan filament:install` to finalize the Filament v3 installation. This command must be run for all new Filament projects.
+
+You can now `composer remove filament/upgrade` as you don't need it any more.
+
+> Some plugins you're using may not be available in v3 just yet. You could temporarily remove them from your `composer.json` file until they've been upgraded, replace them with a similar plugins that are v3-compatible, wait for the plugins to be upgraded before upgrading your app, or even write PRs to help the authors upgrade them.
+
+## Upgrading manually
+
+### High impact changes
+
+#### Config file renamed and combined with other Filament packages
+
+Only one config file is now used for all Filament packages. Most configuration has been moved into other parts of the codebase, and little remains. You should use the v3 documentation as a reference when replace the configuration options you did modify. To publish the new configuration file and remove the old one, run:
+
+```bash
+php artisan vendor:publish --tag=filament-config --force
+rm config/notifications.php
+```
+
+#### New `@filamentScripts` and `@filamentStyles` Blade directives
+
+The `@filamentScripts` and `@filamentStyles` Blade directives must be added to your Blade layout file/s. You can insert `@filamentScripts` after `@livewireScripts` and `@filamentStyles` after `@livewireStyles`.
```

Comment:
Same as above:


I thought Livewire v3 doesn't need these directives anymore, because they are automatically inserted (I only have the information from the Laracon and Laravel News)

https://laravel-news.com/livewire-v3-features

Maybe there will be a Livewire Hook for inserting Scripts/Styles after the Liveries Scripts/Styles?

Reply:
we don't have v3 yet so we don't have a hook. but I hope it exists